qa/tasks/rgw.py

```diff
@@ -153,6 +153,7 @@ def start_rgw(ctx, config, clients):
                 '--rgw_crypt_s3_kms_backend', 'vault',
                 '--rgw_crypt_vault_auth', 'token',
                 '--rgw_crypt_vault_addr', "{}:{}".format(*ctx.vault.endpoints[vault_role]),
+                '--rgw_crypt_vault_prefix', ctx.vault.prefix,
```

Comment:
for rgw config variables that are coming straight from yaml, you can set them directly under `overrides: ceph: conf: client:`. that way we don't have to touch the rgw task to add them on the command line. for an example, see https://github.com/ceph/ceph/pull/30940

Reply:
Nice, I'm rebasing my commit to pick up your changes and I'll add the prefix to the overrides.